File: tools/bench/answers/python_haiku/s2/08_invoice.py
items = [
    ("widget", 4, 2.5),
    ("gizmo", 2, 7.25),
    ("bolt", 10, 0.1)
]
total = 0
max_item = None
max_total = 0
for name, qty, price in items:
    line_total = qty * price
    total += line_total
    if line_total > max_total:
        max_total = line_total
        max_item = name
print(total)
print(max_item)

File: tools/bench/answers/go_haiku/s2/08_invoice.go
package main

import (
	"fmt"
)

func main() {
	items := []struct {
		name  string
		qty   int
		price float64
	}{
		{"widget", 4, 2.5},
		{"gizmo", 2, 7.25},
		{"bolt", 10, 0.1},
	}

	total := 0.0
	maxItem := ""
	maxLineTotal := 0.0

	for _, item := range items {
		lineTotal := float64(item.qty) * item.price
		total += lineTotal
		if lineTotal > maxLineTotal {
			maxLineTotal = lineTotal
			maxItem = item.name
		}
	}

	fmt.Println(total)
	fmt.Println(maxItem)
}
